riscv-asm.md

```diff
@@ -377,6 +377,49 @@ for PIC as seen by objdump:
 			4: R_RISCV_PCREL_LO12_I	.L0
 ```
 
+Load and Store Global
+---------------------
+
+The following pseudo instructions are available to load from and store to
+global objects:
+
+  * `l{b|h|w|d} <rd>, <symbol>`: load byte, half word, word or double word from global
+  * `s{b|h|w|d} <rd>, <symbol>, <rt>`: store byte, half word, word or double word to global
+  * `fl{w|d} <rd>, <symbol>, <rt>`: load float or double from global
+  * `fs{w|d} <rd>, <symbol>, <rt>`: store float or double to global
```

Comment:
This also works for flq/fsq for the Q extension, though I don't know of anyone using that yet.

Reply:
There are flh/fsh instructions for the Zfh half-word float format, but this is still an unofficial draft proposal.